blocks/default-output-block/output-types/default.jsx

```diff
@@ -3,13 +3,14 @@ import { playerRoot, videoOrg } from 'fusion:environment';
 import getProperties from 'fusion:properties';
 import { useFusionContext } from 'fusion:context';
 import { MetaData } from '@wpmedia/engine-theme-sdk';
+import uniqid from './uniqid';
 import './default.scss';
 
 const powaBoot = `${playerRoot}/prod/powaBoot.js?=org=${videoOrg}`;
 const powaDrive = `${playerRoot}/prod/powaDrive.js?org=${videoOrg}`;
 
 const injectStringScriptArray = (scriptStringArray) => scriptStringArray.map((scriptString) => (
-  <script dangerouslySetInnerHTML={{ __html: scriptString }} />
+  <script key={uniqid('script')} dangerouslySetInnerHTML={{ __html: scriptString }} />
```

Comment:
can we find some other way of a key for this? like the key of the env `gaScript` or something like that. We shouldn't need a function like uniqid

Reply:
I think is not so easy. That scripts came from an array on `blocks.json`. They are custom script the site can add.

https://github.com/WPMedia/Fusion-News-Theme/blob/master/blocks.json#L136
https://github.com/WPMedia/fusion-news-theme-blocks/blob/c7da8bccfb8ee711f88274fa1d4f937b42c51340/blocks/default-output-block/output-types/default.jsx#L90

I was thinking on use some kind of hash over the script content but I think overkill for this.

Reply:
ah, I see now. huh, not sure

Reply:
if you remove the key and uniqid script update, I think this pr will be able to remove lots of the warnings based on your before screenshot